src/components/Search/SearchAutocompleteList.tsx

```diff
@@ -203,7 +203,7 @@ function SearchAutocompleteList(
     const [allFeeds] = useOnyx(ONYXKEYS.COLLECTION.SHARED_NVP_PRIVATE_DOMAIN_MEMBER, {canBeMissing: true});
     const cardAutocompleteList = Object.values(allCards);
     const feedAutoCompleteList = useMemo(() => {
-        // We don't want to show the "Expensify Card" feed in the autocomplete suggestion list
+        // We don't want to show the "Expensify Card" feeds in the autocomplete suggestion list
```

Comment:
NAB since this is an existing comment, but it'd be nice to explain why we don't want to show those